website/src/pages/docs/browser-activity.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Inspecting Browser Activity
+---
+
+## Inspecting Browser Activity
+
+In this blog entry, we will use AppView to learn about a browser's network and file activity during a web browsing session.
+
+### Using Firefox and Chrome
+
+Both of these browsers use a sandbox. It is a security feature, intended to isolate the browser processes. Neither of the sandboxes detect the AppView library. The intercept behavior utilized by the library does have undesirable effects on the browser, so we recommend an AppView of the browser without the use of the sandbox.The Chrome sandbox can be disabled from the command line.  The Firefox sandbox is disabled by use of 2 environment variables. 
+
+The detection mechanisms employed by the library are intended to notify about potential security concerns in any given process (we should create a blog describing detection as a precursor to this?). There is behavior used by both browsers that are notified as potential security issues. For example, Chrome OS replaces most standard libc functions by modifying the GOT. The library detects this as a potential issue. It is a viable potential issue in many cases. The library is performing as expected. At the same time, this causes a large number of notifications to be generated. Therefore, these specific notifications can be disabled. Moreover, FIrefox emits a log warning when message queue functions are utilized. The library uses message queue to create a communication channel with an external appview CLI process. If we choose to disable these warnings we disable the use of message queues as an IPC by the library. One other behavior of note from Firefox relates to increased use of physical resources. Firefox processes increase limits on the number of pending signals, for example.  The limit is set to 50,000 and 200,000 max. These seem like large values. Makes sense that library detection would notify about this behavior. These notifications can be disabled if desired.  
```

Comment:
delete: "we should create a blog describing detection as a precursor to this?)."
change FIrefox to Firefox